GOVERNANCE.md

```diff
@@ -31,6 +31,7 @@ individuals and will be valid for 1 year from the start date.
 - VMware [Member: Winnie Kwon; Start Date: 11/10/2022]
 - Ambassador Labs [Member: Richard Li; Start Date: 5/16/2022]
 - Fidelity Investments [Member: Venkat Kasisomayajula / Rajarajan Pudupatti Sundari Jeyakodi; Start Date: 5/16/2022]
+- Tencent Holdings Limited [Member: Xunzhuo Liu; Start Date: 5/16/2022]
```

Comment:
Change the date to the actual start date, 11/17/2022?

Reply:
Thanks! @pydctw : )